tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -1232,6 +1237,9 @@ static const ArBasicKind g_AnyOutputRecordCT[] = {
 static const ArBasicKind g_DxHitObjectCT[] = {AR_OBJECT_HIT_OBJECT,
                                               AR_BASIC_UNKNOWN};
 
+static const ArBasicKind g_VKBufferPointerCT[] = {AR_OBJECT_VK_BUFFER_POINTER,
```

Comment:
We're getting build breaks in Chromium where ENABLE_SPIRV_CODEGEN is not defined. I think this decl should be wrapped in `#ifdef ENABLE_SPIRV_CODEGEN`?

Reply:
I'm working on a patch.

Reply:
Created [pull request](https://github.com/microsoft/DirectXShaderCompiler/pull/7306).